CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ This changelog keeps track of work items that have been completed and are ready
 - **Operator**: Remove ScaledObject `name` & `app` custom labels ([#717](https://github.com/kedacore/http-add-on/issues/717))
 - **Scaler**: Provide graceful shutdown for grpc server on SIGINT and SIGTERM ([#731](https://github.com/kedacore/http-add-on/issues/731))
 - **Scaler**: remplement custom interceptor metrics ([#718](https://github.com/kedacore/http-add-on/issues/718))
+- **Interceptor**: Add support for streaming responses ([#726](https://github.com/kedacore/http-add-on/issues/743))
```

Comment:
Please, sort this alphabetically and update the id too
```suggestion
- **Interceptor**: Add support for streaming responses ([#742](https://github.com/kedacore/http-add-on/issues/https://github.com/kedacore/http-add-on/issues/742))
```

Reply:
fixed